src/scheduler/data_types.h

```diff
@@ -756,6 +756,7 @@ struct resv_info
 	int resv_idx;			/* the index of standing resv occurrence */
 	int count;			/* the total number of occurrences */
 	time_t req_start;		/* user requested start time of resv */
+	time_t req_start_future;	/* For standing reservations being modified, this will be the start time of future occurrences */
```

Comment:
I'd call this req_start_orig.  The comment is wrong too.  This isn't the start of the future occurrences, this is used to create the start time of the future occurrences.